### PR TITLE
Feature/#132 restaurant list apply design

### PR DESCRIPTION
**Pull request by manudous**
@brauliodiez

### CODE REVIEW

File: menu-public-portal/src/pods/restaurant-list/restaurant-list.container.tsx [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
-  { name: "Hermanos Alba", urlName: "hermanosalba" },
-];
 
 const RestaurantListContainer: React.FC = () => {
+  const [restaurantList, setRestaurantList] = React.useState<RestaurantInfo[]>(
+    []
+  );
+
+  React.useEffect(() => {
+    getRestaurantList().then(setRestaurantList);
```
Comment (brauliodiez) [inline menu-public-portal/src/pods/restaurant-list/restaurant-list.container.tsx:12]: This cannot be placed in the container, we are using SSR, Container needs to have a prop restaurantCollection

Comment (manudous) [reply menu-public-portal/src/pods/restaurant-list/restaurant-list.container.tsx]: done

File: menu-public-portal/src/pages/index.tsx [hunk 2]
```diff
@@ -1,7 +1,24 @@
-import type { NextPage } from "next";
+import type { GetStaticProps, NextPage } from "next";
 import { RestaurantListContainer } from "pods/restaurant-list";
-const Home: NextPage = () => {
-  return <RestaurantListContainer />;
+import { RestaurantInfo } from "pods/restaurant-list/restaurant-list.vm";
+import { getRestaurantList } from "pods/restaurant-list/";
+
+export const getStaticProps: GetStaticProps = async () => {
+  const restaurantList: RestaurantInfo[] = await getRestaurantList();
```
Comment (brauliodiez) [inline menu-public-portal/src/pages/index.tsx:7]: RestaurantCollection

File: menu-public-portal/src/pods/restaurant-list/restaurant-list.container.tsx [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
-];
 
-const RestaurantListContainer: React.FC = () => {
+interface Props {
+  restaurantList: RestaurantInfo[];
+}
+
+const RestaurantListContainer: React.FC<Props> = (props) => {
+  const { restaurantList } = props;
   return <RestaurantList restaurantList={restaurantList} />;
```
Comment (brauliodiez) [inline menu-public-portal/src/pods/restaurant-list/restaurant-list.container.tsx:11]: restaurantCollection

**Pull request merged**